tools/wake/main.cpp

```diff
@@ -68,178 +68,171 @@
-    { 'm', "memory",                GOPT_ARGUMENT_REQUIRED  | GOPT_ARGUMENT_NO_HYPHEN },
-    { 'c', "check",                 GOPT_ARGUMENT_FORBIDDEN },
-    { 'v', "verbose",               GOPT_ARGUMENT_FORBIDDEN | GOPT_REPEATABLE },
-    { 'd', "debug",                 GOPT_ARGUMENT_FORBIDDEN },
-    { 'q', "quiet",                 GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "no-wait",               GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "no-workspace",          GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "no-tty",                GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "fatal-warnings",        GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "heap-factor",           GOPT_ARGUMENT_REQUIRED  | GOPT_ARGUMENT_NO_HYPHEN },
-    { 0,   "profile-heap",          GOPT_ARGUMENT_FORBIDDEN | GOPT_REPEATABLE },
-    { 0,   "profile",               GOPT_ARGUMENT_REQUIRED  },
-    { 'C', "chdir",                 GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "in",                    GOPT_ARGUMENT_REQUIRED  },
-    { 'x', "exec",                  GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "job",                   GOPT_ARGUMENT_REQUIRED  },
-    { 'i', "input",                 GOPT_ARGUMENT_FORBIDDEN },
-    { 'o', "output",                GOPT_ARGUMENT_FORBIDDEN },
-    { 'l', "last",                  GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "lsp",                   GOPT_ARGUMENT_FORBIDDEN },
-    { 'f', "failed",                GOPT_ARGUMENT_FORBIDDEN },
-    { 's', "script",                GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "init",                  GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "version",               GOPT_ARGUMENT_FORBIDDEN },
-    { 'g', "globals",               GOPT_ARGUMENT_FORBIDDEN },
-    { 'e', "exports",               GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "html",                  GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "timeline",              GOPT_ARGUMENT_OPTIONAL  },
-    { 'h', "help",                  GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "debug-db",              GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "stop-after-parse",      GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "stop-after-type-check", GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "stop-after-ssa",        GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "no-optimize",           GOPT_ARGUMENT_FORBIDDEN },
-    { 0,   "tag-dag",               GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "tag",                   GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "export-api",            GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "stdout",                GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "stderr",                GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "fd:3",                  GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "fd:4",                  GOPT_ARGUMENT_REQUIRED  },
-    { 0,   "fd:5",                  GOPT_ARGUMENT_REQUIRED  },
-    { ':', "shebang",               GOPT_ARGUMENT_REQUIRED  },
-    { 0,   0,                       GOPT_LAST}};
+    {'p', "percent", GOPT_ARGUMENT_REQUIRED | GOPT_ARGUMENT_NO_HYPHEN},
```

Comment:
something is bugged here

Reply:
Yeah, looks like the auto formatting is having issues with this block of code.

It's to be expected that we'll have some cases like this so I just turned off formatting for this section and manually formatted it.

We will like have several of these that look better hand formatted vs what the auto formatter does.

(While I was hear, I skipped formatting for the help message as well, makes it a little bit easier to read)